packages/mds-compliance-service/repository/index.ts

```diff
@@ -29,6 +29,7 @@ import {
   GetComplianceViolationsByTimeIntervalOptions
 } from '../@types'
 import { ComplianceSnapshotEntity } from './entities/compliance-snapshot-entity'
+import { ComplianceSnapshotKafkaFailureEntity } from './entities/compliance-snapshot-kafka-failure'
```

Comment:
BTW, Can we rename the entity to just `ComplianceSnapshotFailureEntity`. There isn't anything Kafka specific about this table and it seems like it could be used more generically for other types of compliance snapshot failures. Maybe could add a failure type column or something.